src/main-content/main-content.css

```diff
@@ -0,0 +1,14 @@
+:root {
+  --omrs-sidenav-width: 16.25rem;
+  --omrs-topnav-height: 3.5rem;
+}
+
+.omrs-main-content {
+  margin-top: var(--omrs-topnav-height);
+  margin-left: 0;
+  transition: margin-left 0.5s ease;
+}
+
+body.omrs-sidenav-expanded .omrs-main-content {
```

Comment:
See related https://github.com/openmrs/openmrs-esm-primary-navigation/pull/2 for how `omrs-sidenav-expanded` is placed on `document.body`.